src/Compilers/CSharp/Portable/Binder/ImportChain.cs

```diff
@@ -23,11 +23,6 @@ public ImportChain(Imports imports, ImportChain parentOpt)
             ParentOpt = parentOpt;
         }
 
-        private string GetDebuggerDisplay()
-        {
-            return $"{Imports.GetDebuggerDisplay()} ^ {ParentOpt?.GetHashCode() ?? 0}";
-        }
-
```

Comment:
This is not dead code.  It's referenced in the type attribute and used at debug time.  Same for every other GetDebuggerDisplay that is referenced.